docs/index.md

```diff
@@ -1,41 +1,40 @@
-# Starting on the DAaaS Platform
+# Starting on the Advanced Analytics Workspace
 
-![The DAaaS Platform](images/readme/portal_ui.png)
+![Advanced Analytics Workspace homepage](images/readme/portal_ui.png)
 
-**Link to the portal [here](portal.example.ca)**
+The **[Advanced Analytics Workspace portal](portal.covid.cloud.statcan.ca)** is
```

Comment:
Not sure if we're releasing the domain yet. Will let @sylus confirm.

Also, need to prefix urls with `https://` or else they get interpreted wrong